addons/mail_group/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Mail Group",
+    'summary': """
+        Manage your mailing lists from Odoo.
+    """,
+    'description': """
+        Manage your mailing lists from Odoo.
+    """,
+    'version': '1.0',
+    'depends': ['mail', 'portal'],
+    'auto_install': True,
```

Comment:
Should it actually be auto-install? I'm thinking a lot of people will never use that feature.

Reply:
It should because otherwise during the migration you might loose your channel with "email_send=True" and the related message

Reply:
Migration should be handled differently than new db though, and the manifest is mainly done for new db.

Reply:
@seb-odoo So in my migration script I should add "auto_install=True" (to install the module is we can), and in the manifest keep it at False ?

Reply:
Oh indeed, it worked because it has access to the `mail.group.message` but I should update them anyway

Reply:
I'm not expert on migration for new modules, I've actually added the comment here after I saw that you did find an instruction to auto install in the migration script. So if that works as intended, then I guess yes that it how we should do it.

Reply:
@seb-odoo Just tested, and `auto_install=False` in the manifest but `auto_install=True` in the migration script install the module during the migration

Reply:
I would not make it auto install at migration: I would install it only if people have email-based channels. If they don't I would not install it as they don't use the feature, so no need to install it.